src/ray/common/status.h

```diff
@@ -138,6 +139,13 @@ class RAY_EXPORT Status {
   Status(const Status &s);
   void operator=(const Status &s);
 
+  // Support abseil string formatting function, for example, `absl::StrCat`,
+  // `absl::StrFormat`, etc.
+  template <typename Sink>
+  friend void AbslStringify(Sink &sink, const Status &status) {
+    sink.Append(status.ToString());
+  }
```

Comment:
Lets add it later when it's actually needed. In this PR, its only used in test.

Reply:
Emm stringify is needed for printing. For example, one thing I'm thinking is to write a generalized printer which accepts all types.

One use case is check: https://github.com/ray-project/ray/blob/265dcc60a4bd279eb3bedfc94308519877f7fdb6/src/ray/util/logging.h#L154

It should be updated to
```
RAY_CHECK((_left_ op _right_)) << " " << DebugString(_left_) << " vs " << DebugString(_right_)
```
otherwise it doesn't accept non-ostream-printable types.

Reply:
Let's have this change when you actually do `to write a generalized printer which accepts all types.`? Otherwise the code is basically dead code now.